sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+# SIP-028: sBTC Bootstrap
+
+- **SIP Number:** 028
+- **Title:** A Decentralized and Programmable Asset Backed 1:1 with BTC 
+- **Consideration:** Technical, Economics
+- **Type:** Informational
+- **Status:** Draft
+- **Authors:**
+  - Andre Serrano
+  - Ashton Stephens
+  - Joey Yandle
+  - Mårten Blankfors
+  - Jesus Najera
+  - Jude Nelson
+  - Friedger Müffke
+  - Tycho Onnasch
+
+## Abstract
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers such that only a 70% majority can access the funds to maintain the protocol.
+
+While Bitcoin on its own cannot be used with smart contracts, sBTC provides a bridge of value from the Bitcoin Blockchain to the Stacks Blockchain to enable users who only own BTC to utilize the capability of smart contracts without switching to a new currency. This SIP aims to describe the high level sBTC system and the criteria for signer selection. 
+
+This first iteration of sBTC is purely at the application layer and does not affect network consensus or operation. This SIP does not attempt to describe the technical details of this or any subsequent sBTC releases; it only seeks to get community approval for the requirements of the sBTC bootstrap signers.
+
+## Introduction
+
+### Glossary
+
+| Term               | Definition                                                                                              |
+|--------------------|---------------------------------------------------------------------------------------------------------|
+| SIP-10 Token       | A token on the Stacks blockchain that adheres to the fungible token standards outlined in [SIP-10](https://github.com/stacksgov/sips/blob/main/sips/sip-010/sip-010-fungible-token-standard.md).        |
+| sBTC               | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
```

Comment:
Pervasively, "Bitcoin Blockchain" is a typo.  "Blockchain" is not capitalized here.